src/serverenvironment.cpp

```diff
@@ -1603,14 +1603,12 @@ void ServerEnvironment::getSelectedActiveObjects(
 	const core::line3d<f32> &shootline_on_map,
 	std::vector<PointedThing> &objects)
 {
-	std::vector<u16> objectIds;
+	std::vector<ServerActiveObject *> objectIds;
```

Comment:
should be renamed to `objects` too

Reply:
renamed to objs, we have objects as a parameter already